substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -169,8 +181,8 @@ impl<T: Config> Pallet<T> {
 
 		ledger.total = ledger.total.checked_add(&extra).ok_or(ArithmeticError::Overflow)?;
 		ledger.active = ledger.active.checked_add(&extra).ok_or(ArithmeticError::Overflow)?;
-		// last check: the new active amount of ledger must be more than ED.
-		ensure!(ledger.active >= asset::existential_deposit::<T>(), Error::<T>::InsufficientBond);
+		// last check: the new active amount of ledger must be more than min bond.
+		ensure!(ledger.active >= Self::min_bond(), Error::<T>::InsufficientBond);
```

Comment:
Best to make sure the error docs for InsufficientBond, which will be shown in UIs, contains good info.